src/components/card/CardSection.tsx

```diff
@@ -36,7 +36,6 @@ export type CardSectionProps = ViewProps & {
    * Will be used for the background when provided
    */
   imageSource?: ImageSourcePropType;
-  source?: ImageSourcePropType; // TODO: remove after deprecation
```

Comment:
I found a usage in OneApp that still use the `source` prop. 
https://github.com/wix-private/restaurants/blob/af89e6db06c8e694e4d38719466ec88a93d5ffdc/one-app-restaurants-member/src/orders/screens/DispatchTypeScreen/components/DispatchTypeComponent.tsx#L51

Reply:
That's wired. The change was already in the first V6 branch...

Reply:
I asked @guywix to fix it.

Reply:
Did he respond? 
Does he have an ETA? 
We can still release v6 to community and Guy will have time till our next private version (which will take v6)

Reply:
He didn't, but that's what I though as we didn't plan to release in private just yet

Reply:
Ok, but it's important to remember not to miss it. 
Anyway, ill approve